### PR TITLE
enh(release): Add Centreon Web 20.04.19 RN

### PR DESCRIPTION
**Pull request by lpinsivy**
## Description

Please include a short resume of the changes and what is the purpose of the PR.

Any relevant information should be added to help reviewers.

## Target serie

- [x] 20.04.x
- [ ] 20.10.x
- [ ] 21.04.x
- [ ] 21.10.x (master)

### CODE REVIEW

**Review by thiuyendang — APPROVED** [review]

File: fr/releases/centreon-core.md [hunk 1]
```diff
@@ -18,6 +18,17 @@ commerciales, vous pouvez vous rendre sur notre
...
+
+#### Enhancement
+
+- Compatibility with PHP 7.3
+
+#### Security fixes
+
+- Fixed session on account deletion
+
+
```
Comment (mdebauchez) [inline fr/releases/centreon-core.md:31]: There is one line break too much

**Review by mdebauchez — CHANGES_REQUESTED** [review]

**Review by mdebauchez — COMMENTED** [review]
There is one line break too much at lign 31 in french

**Review by mdebauchez — APPROVED** [review]

File: en/releases/centreon-core.md [hunk 2]
```diff
@@ -19,6 +19,16 @@ If you have feature requests or want to report a bug, please go to our
 
 ## Centreon Web
 
+### 20.04.19
+
```
Comment (tuntoja) [inline en/releases/centreon-core.md:23]:
```suggestion

`September 30, 2021`

```

File: fr/releases/centreon-core.md [hunk 3]
```diff
@@ -18,6 +18,16 @@ commerciales, vous pouvez vous rendre sur notre
 
 ## Centreon Web
 
+### 20.04.19
+
```
Comment (tuntoja) [inline fr/releases/centreon-core.md:22]:
```suggestion

`September 30, 2021`

```

**Review by tuntoja — CHANGES_REQUESTED** [review]

**Review by tuntoja — APPROVED** [review]

**Pull request merged**